src/Compilers/CSharp/Test/Semantic/Semantics/FunctionPointerTests.cs

```diff
@@ -47,6 +57,74 @@ public void UsingAliasTest()
                 Diagnostic(ErrorCode.HDN_UnusedUsingDirective, "using s = delegate*<void>;").WithLocation(2, 1));
         }
 
+        [Fact]
+        public void UsingAliasTest_UnsafeModifier()
+        {
+            var src = @"
+using unsafe S = delegate*<void>;";
+
+            var comp = CreateCompilationWithFunctionPointers(src, options: TestOptions.UnsafeDebugDll, parseOptions: TestOptions.Regular11);
```

Comment:
>TestOptions.Regular11 [](http://example.com/codeflow?start=117&length=21)

If it is worth testing this scenario for version 11, then, I think, it is worth testing for version 12 as well.

Reply:
The comment is applicable to other tests as well